mayastor/src/bin/mayastor.rs

```diff
@@ -60,7 +63,30 @@ fn start_tokio_runtime(args: &MayastorCliArgs) {
     });
 }
 
-const VERSION_STR: &str = "0.9.0 C2.RC2";
+fn hugepage_check() {
+    let hugepage_path = Path::new("/sys/kernel/mm/hugepages/hugepages-2048kB");
+    let nr_pages: u32 = sysfs::parse_value(&hugepage_path, "nr_hugepages")
+        .expect("failed to read the number of pages");
+    let free_pages: u32 = sysfs::parse_value(&hugepage_path, "free_hugepages")
+        .expect("failed to read the number of free pages");
+    if nr_pages < PAGES_NEEDED {
+        error!(
+            "insufficient huge pages allocated, needed:  {}  allocated: {}",
+            PAGES_NEEDED, nr_pages
+        );
+        std::process::exit(1)
+    }
+
+    if free_pages < PAGES_NEEDED {
+        error!(
+            "not enough free pages available: {}, free: {}",
```

Comment:
Wording is inconsistent and maybe a bit confusing, how about `insufficient huge pages free, needed: {}, free: {}`.